### PR TITLE
Add timeout to terraform import command

### LINKED ISSUE #933: [bug] [GCP] QHub Deploy hangs on terraform import step when bucket doesn't exist
## Describe the bug

When running `qhub deploy -c qhub-config.yaml` for the first time on a new cluster on GCP, the script hangs when trying to refresh the state of the `terraform-state` bucket. The terraform logs via`TF_LOG=TRACE` show that this is because the bucket does not exist. It will continually check the state of the bucket every 10 seconds, thus being the cause of the terminal hanging.

This can be reproduced by rendering a qhub-config.yaml, changing directories into `terraform state` and then running `terraform import module.terraform-state.module.gcs.google_storage_bucket.static-site <project name>-dev-terraform-state` replacing <project name> with the relevant project name

If the bucket is manually created by running `terraform apply --auto-approve` inside of `terraform state`, the deployment is able to complete normally.

### PR DESCRIPTION
**Pull request by tylerpotts**
Fixes #933

## Changes:

Adds a timeout to the subprocess for `terraform import` of 30 seconds. Without this timeout deploying on GCP will hang and give the error that that the bucket doesn't exist

## Types of changes

What types of changes does your code introduce?

_Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

### Requires testing

- [ ] Yes
- [x] No

### In case you checked yes, did you write tests?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by viniciusdc — APPROVED** [review]
LGTM!

**Comment by danlester** [comment]
Actually this doesn't quite work - I'm taking a look at fixing it.

**Comment by danlester** [comment]
Sorry if it was a WIP, but thought it was just waiting for a Black fix before it could be merged...

**Comment by danlester** [comment]
Trying to fix again in https://github.com/Quansight/qhub/pull/955

**Pull request merged**
